spec/acceptance/candlepin_spec.rb

```diff
@@ -163,4 +163,49 @@ class { 'certs::candlepin':
     its(:stdout) { should match(/^Owner: CN=localhost, OU=SomeOrgUnit, O=Katello, ST=North Carolina, C=US$/) }
     its(:stdout) { should match(/^Issuer: CN=#{host_inventory['fqdn']}, OU=SomeOrgUnit, O=Katello, L=Raleigh, ST=North Carolina, C=US$/) }
   end
+
+  context 'updates java-client certificate in truststore if it changes' do
+    let(:pp) do
+      <<-EOS
+      user { 'tomcat':
+        ensure => present,
+      }
+
+      ['/usr/share/tomcat/conf', '/etc/candlepin/certs'].each |$dir| {
+        exec { "mkdir -p ${dir}":
+          creates => $dir,
+          path    => ['/bin', '/usr/bin'],
+        }
+      }
+
+      package { 'java-1.8.0-openjdk-headless':
+        ensure => installed,
+      }
+
+      include certs::candlepin
+      EOS
+    end
+
+    before(:all) do
+      pp_setup = <<-EOS
+        exec { "rm -rf /root/ssl-build/localhost":
+          path => "/bin:/usr/bin",
+        }
+      EOS
+
+      apply_manifest(pp_setup, catch_failures: true)
```

Comment:
You don't need to do this via Puppet, I think this also works:
```suggestion
  on default, 'rm -rf /root/ssl-build/localhost'
```